### DISCUSSION TITLE
IVR Speech Recognition Question

### DISCUSSION THREAD

**Discussion by csharpcoder198** [shinyorg/shiny#1007] [Ideas]
I have a feature request to implement IVR (Interactive Voice Recognition)  Notifications which are spoken and Interactively voice responded.

Shiny is a big library. Is that functionality inside the box? Is there other libraries that I should consider for this

**Comment by aritchie** [comment]
This isn’t something Shiny offers at all.  This may be possible through the OS platform with accessibility turned on but I’m not sure
